CaesarCipher/CaesarCipher/main.m

```diff
@@ -12,6 +12,7 @@ @interface CaesarCipher : NSObject
 
 - (NSString *)encode:(NSString *)string offset:(int)offset;
 - (NSString *)decode:(NSString *)string offset:(int)offset;
+- (NSString *)codeBreaker: (NSString *) string_one, (NSString *) string_two;
```

Comment:
This syntax isn't correct. Take a look at this post http://blog.teamtreehouse.com/the-beginners-guide-to-objective-c-methods